### ISSUE TITLE
Inconsistent naming of 'look_' glyphs

### ISSUE THREAD

**Issue by s9j4n8dba** [google/material-design-icons#797]
For some reason the glyphs whose name begin with 'looks_' are called 'looks_one', 'looks_two', 'looks_3', 'looks_4', 'looks_5' and 'looks_6'. Using numerals for every glyph would make more sense.

**Comment by RoelN** [comment]
An alias could be added to the font, so `looks_1` and `looks_one` point to the same glyph.

Noted, thanks for reporting!

**Comment by tphinney** [comment]
Yes, we don’t want to _change_ the name for stability reasons—but we can add an additional ligature that points at the same glyph, as if it had the other name.

So if we add ligatures for “looks_1” and “looks_2” that should do it, right?
